app/Role.php

```diff
@@ -15,6 +15,11 @@ class Role extends Model
     const GUSET = 5;
     const NETWORK_COORDINATOR = 6;
 
+    const REPAIR_DIRECTORY_NONE = 0;
```

Comment:
Will this set the Repair Directory role to 0 if they are set as having no access?  I think this will cause an error on the Repair Directory Doctrine mapping, as it will try and find a role entity with id of 0.  Whereas if repairdir_role is null, it will ignore it.

Reply:
We do map - see below.